x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -1235,6 +1263,16 @@ var security = (function () {
         .Add(addActionDesc)
         .Build();
 
+    var event4688 = new processor.Chain()
+        .Add(copySubjectUser)
+	.Add(renameNewProcessFields)
+        .Build();    
```

Comment:
In the sysmon module we add an `event.category` and `event.type` for event ID 1 and 5. To be consistent I'd apply those same values here. We might have to change them in the future when ECS finalizes on a set of categories.

https://github.com/elastic/beats/blob/ca81776033889ad59bc3b571e58ea1a90fba4f5e/x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js#L396-L397
https://github.com/elastic/beats/blob/ca81776033889ad59bc3b571e58ea1a90fba4f5e/x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js#L491-L492